client/client.go

```diff
@@ -470,7 +470,7 @@ func (c *Client) Authenticate() error {
 	if obj == nil {
 		return errors.New("Empty response")
 	}
-	err = CheckForErrors(obj, method, c.skipLoggingPayload)
+	err = CheckForErrors(obj, method, true)
```

Comment:
Good catch on this one. I think we were printing too much in the past.

Can you have a look in CheckForErrors as there are some logs that we now will not print but we could have an else version without the "cont" variable displayed?

Reply:
Yeah, it looks like it was printing the whole response from APIC on errors including HTTP 401/403.
I don't think APIC responses would be leaking sensitive info but it wouldn't really be necessary here.
Even if we skip logging the function still returns errors like, `Unable to authenticate. Please check your credentials` or in the case of other errors it still returns the actual error message from APIC.